### PR TITLE
Monsters Update and Fix (melee effect & targetDistance)

### LINKED ISSUE #1968: BUG:  Monsters still attacking from range when should be melee range
## READ BEFORE PROCEEDING: OUR SERVER IS UPDATED DAILY. IF POSSIBLE MAKE SURE TO UPDATE TO THE LATEST VERSION BEFORE REPORTING.

Hero cave, old fortress, sea serpent, death portal etc

### PR DESCRIPTION
**Pull request by heberholanda**
Thanks @LIKEAHYENA for fix

- [x] Melee effect
- [x] TargetDistance
- [x] Others monsters Updated

Thanks @Xiolones for fix
- [x] Amphibics, Aquatics, Birds and Bosses Updated

target1
target2

### CODE REVIEW

File: src/monsters.cpp [hunk 1]
```diff
@@ -873,7 +873,7 @@ MonsterType* Monsters::loadMonster(const std::string& file, const std::string& m
 			} else if (strcasecmp(attrName, "lightcolor") == 0) {
 				mType->info.light.color = pugi::cast<uint16_t>(attr.value());
 			} else if (strcasecmp(attrName, "targetdistance") == 0) {
-				mType->info.targetDistance = std::max<int32_t>(1, pugi::cast<int32_t>(attr.value()));
```
Comment (debone) [inline src/monsters.cpp]: Why did you remove from XML? :P

**Comment by gabz182** [comment]
Tested, working!
Thank you.

**Review by dudantas — APPROVED** [review]
We appreciate your contribution! Thanks!

**Comment by majestyotbr** [comment]
Updated PR description.

Before:
```
Thanks @LIKEAHYENA for fix

- [x] Melee effect 

- [x] TargetDistance

- [x] Others monsters Updated

- [x] Amphibics and Aquatics Updated

- [x] Bosses Update

![target1](https://user-images.githubusercontent.com/715547/99213952-b71eb300-27ad-11eb-8272-65e7b7456e22.gif)
![target2](https://user-images.githubusercontent.com/715547/99213958-ba19a380-27ad-11eb-886b-b4d68560e9bf.gif)
```

After:
```
Thanks @LIKEAHYENA for fix

- [x] Melee effect 

- [x] TargetDistance

- [x] Others monsters Updated

Thanks @Xiolones for fix

- [x] Amphibics, Aquatics, Birds and Bosses Updated

![target1](https://user-images.githubusercontent.com/715547/99213952-b71eb300-27ad-11eb-8272-65e7b7456e22.gif)
![target2](https://user-images.githubusercontent.com/715547/99213958-ba19a380-27ad-11eb-886b-b4d68560e9bf.gif)
```

Resolved.

**Pull request merged**